src/features/TodosViewForm.jsx

```diff
@@ -0,0 +1,29 @@
+
+function TodosViewForm ({sortDirection, setSortDirection, sortField, setSortField, queryString, setQueryString  }){
+    function preventRefresh(event){
+        event.preventDefault();
+    }
+
+    return(
+        
+        <div>
+            <div>
+            <label htmlFor="search">Search todos:</label>
+            <input id="search" type="text"
+            value={queryString}
+            onChange={(e) => setQueryString(e.target.value)} />
+            <button type="button" onClick={ () => setQueryString('') }>Clear</button>
+            </div>
+
+            <form action="" onSubmit={preventRefresh} >
+                <label htmlFor="view">Sort by</label>
+                <select name="" id="view" onChange={ (e) => setSortField (e.target.value)} value= {sortField}>
+                    <option value="title">Title</option>
+                    <option value="createdTime">Time added</option>
+                </select>
+            </form>
```

Comment:
You'll want to add a label containing "Direction" on line 24, just after the previous label and select. 

Then you'll want to add another select with a `value` of `sortDirection` and `option` elements for: 
    • "Ascending", using "asc" for the value 
    • "Descending", using "desc" for the value 
    
You’ll want its `onChange` to take the event object and call `setSortDirection` with the event target's value.